app/models/post.rb

```diff
@@ -0,0 +1,3 @@
+class Post < ApplicationRecord
+  belongs_to :user
+end
```

Comment:
- [ ] As I already pointed out that it would be great if the complete the association by adding the has_many part in the Post model to make your association work according to the project requirements :+1:
- [ ] It would be nice if you double-check the database and then you would find that post has a foreign key author_id which connects the post table to the user table so it would be great if you fix this problem as well by mentioning the author in the belongs_to in this model :+1: Kindly see the below snippet to get an idea what I am talking about :blush:

```ruby
belongs_to :author, class_name: 'User'
has_many :comments
has_many :likes
```